### PR TITLE
Add a shoot at player (hard) function

### PR DESCRIPTION
**Pull request by khaneh321**
Added a shootAtPlayer_hard() function that should shoot the player's grid at random locations. This function is currently not being called and should be used once inheritance is properly implemented.

### CODE REVIEW

File: src/main/java/edu/oregonstate/cs361/battleship/BattleshipModel.java [hunk 1]
```diff
@@ -142,6 +142,20 @@ public void shootAtPlayer() {
...
+        //this is hard mode, shoots at random locations
+        int max = 10;
+        int min = 1;
+        Random random = new Random();
+        int randRow = random.nextInt(max - min + 1) + min;
+        int randCol = random.nextInt(max - min + 1) + min;
+        Coordinate coor = new Coordinate(randRow,randCol);
+        playerShot(coor);
+        //end of hard mode
+    }
```
Comment (munishaparikh) [inline src/main/java/edu/oregonstate/cs361/battleship/BattleshipModel.java:155]: This is good what you have added for the hard function.

File: src/main/java/edu/oregonstate/cs361/battleship/Main.java [hunk 2]
```diff
@@ -65,7 +65,7 @@ private static String fireAt(Request req) {
         int colInt = Integer.parseInt(col);
         currModel.shootAtComputer(rowInt,colInt);
         currModel.shootAtPlayer();
-
+        //currModel.shootAtPlayer_hard();
```
Comment (munishaparikh) [inline src/main/java/edu/oregonstate/cs361/battleship/Main.java:68]: This function should be called. So we should figure how to call it using inheritence

**Comment by munishaparikh** [comment]
This Works!

File: src/main/java/edu/oregonstate/cs361/battleship/BattleshipModel.java [hunk 3]
```diff
@@ -142,6 +142,20 @@ public void shootAtPlayer() {
             //end of easy mode
     }
 
+    public void shootAtPlayer_hard() {
+        //this is hard mode, shoots at random locations
+        int max = 10;
+        int min = 1;
+        Random random = new Random();
+        int randRow = random.nextInt(max - min + 1) + min;
```
Comment (alfayeza) [inline src/main/java/edu/oregonstate/cs361/battleship/BattleshipModel.java:150]: This is good. Although you could have done it differently. You can use for loop to do the same thing.

**Pull request merged**